catalogue/webapp/components/PhysicalItems/PhysicalItems.tsx

```diff
@@ -98,6 +98,7 @@ const PhysicalItems: FunctionComponent<Props> = ({
       const updateItemsStatus = async () => {
         const itemsResponse = await fetch(`/api/works/items/${work.id}`, {
           signal,
+          credentials: 'same-origin',
```

Comment:
Comment about why this is here / that it's not actually related to credentials?

Reply:
Added.